app/assets/stylesheets/widgets/feeds/process.scss

```diff
@@ -0,0 +1,15 @@
+.feed-processes {
+
+  .sdg-tag-list {
+    padding: 6px;
+
+    .more-goals,
+    .more-targets {
+      color: $text;
```

Comment:
If we keep the `.more-*` tags, how about moving the styles to the `tag_list` SCSS file?

```diff
diff --git a/app/assets/stylesheets/sdg/tag_list.scss b/app/assets/stylesheets/sdg/tag_list.scss
index 2a0018a1f7..09875d2de5 100644
--- a/app/assets/stylesheets/sdg/tag_list.scss
+++ b/app/assets/stylesheets/sdg/tag_list.scss
@@ -12,7 +12,7 @@
     @extend %tags;
 
     a:not(.more-targets),
-    span {
+    span:not(.more-targets) {
       color: $white;
     }
 
diff --git a/app/assets/stylesheets/widgets/feeds/process.scss b/app/assets/stylesheets/widgets/feeds/process.scss
index c7d3095263..5d05068375 100644
--- a/app/assets/stylesheets/widgets/feeds/process.scss
+++ b/app/assets/stylesheets/widgets/feeds/process.scss
@@ -5,7 +5,6 @@
 
     .more-goals,
     .more-targets {
-      color: $text;
 
       &:hover {
         background: #ececec;
```